bin/cdxgen.js

```diff
@@ -141,6 +148,7 @@ const args = yargs(hideBin(process.argv))
   })
   .option("fail-on-error", {
     type: "boolean",
+    default: !isSecureMode,
```

Comment:
This was a mistake. This made `fail-on-error` the default for everyone. Apologies.